packages/api/src/command/medical/document/document-query.ts

```diff
@@ -64,38 +57,45 @@ export async function queryDocumentsAcrossHIEs({
       }
     );
   }
-  const docQueryProgress = patient.data.documentQueryProgress;
+
+  // Doc Request should be undefined here, since a new DocRequest is created each time the endpoint is called.
+  const docQueryProgress = docRequest.documentQueryProgress;
+  console.log("Doc Request Object: ", docQueryProgress);
   const requestId = getOrGenerateRequestId(docQueryProgress);
 
   if (
     docQueryProgress?.download?.status === "processing" ||
     docQueryProgress?.convert?.status === "processing"
   ) {
     log(`Patient ${patientId} documentQueryStatus is already 'processing', skipping...`);
-    return createQueryResponse("processing", patient);
+    return createQueryDocRequestResponse("processing", docRequest);
   }
 
   const externalData = patient.data.externalData?.COMMONWELL;
-  if (!externalData) return createQueryResponse("failed");
+  if (!externalData) return createQueryDocRequestResponse("failed");
 
   const cwData = externalData as PatientDataCommonwell;
-  if (!cwData.patientId) return createQueryResponse("failed");
+  if (!cwData.patientId) return createQueryDocRequestResponse("failed");
 
-  const updatedPatient = await updateDocQuery({
+  const [updatedDocRequest, updatedPatient] = await appendDocRequestQueryProgress({
     patient: { id: patient.id, cxId: patient.cxId },
+    docRequest,
     downloadProgress: { status: "processing" },
     requestId,
     reset: true,
   });
 
+  // TODP: is there anything to do with updated patient
+  console.log("Updated Patient: ", JSON.stringify(updatedPatient));
+
   getDocumentsFromCW({
```

Comment:
This portion has yet to be updated. Does getDocumentsFromCW need the docRequest too in order to update the progress appropriately?